sdoapp.py

```diff
@@ -1062,11 +1062,10 @@ def handleHomepage(self, node):
         # print "accept_header: " + str(accept_header) + " mimereq: "+str(mimereq) + "Scores H:{0} XH:{1} J:{2} ".format(html_score,xhtml_score,jsonld_score)
 
         if (ENABLE_JSONLD_CONTEXT and (jsonld_score < html_score and jsonld_score < xhtml_score)):
-            jsonldcontext = GetJsonLdContext(layers=ALL_LAYERS)
-            self.response.headers['Content-Type'] = "application/ld+json"
-            #self.emitCacheHeaders()
```

Comment:
I did and it only half fixed it.

It stopped the context appearing instead of html, but it didn’t stop the context being pulled into json-ld playground being the cached version of the html.

Fixed it by 302 redirecting to the json-ld context file and not supplying etag on the html version of the page.